src/EnergyPlus/CMakeLists.txt

```diff
@@ -382,8 +382,8 @@ SET( SRC
   OutAirNodeManager.hh
   OutdoorAirUnit.cc
   OutdoorAirUnit.hh
-  OutputFiles.cc
-  OutputFiles.hh
+        IOFiles.cc
+        IOFiles.hh
```

Comment:
Indent fixes here would be good if anything else needs changing, otherwise it is fine.